service/share/manage.go

```diff
@@ -20,6 +20,7 @@ type (
 	ShareCreateService struct {
 		Uri             string `json:"uri" binding:"required"`
 		IsPrivate       bool   `json:"is_private"`
+		Password        string `json:"password"`
```

Comment:
Considering validation rule: `a-zA-Z0-9`. Other special characters might require percentage encoding, and that's a pain in the ass we should avoid.